src/gui/cmdwidget.cpp

```diff
@@ -0,0 +1,61 @@
+#include "cmdwidget.h"
+
+namespace NeovimQt {
+
+CmdWidget::CmdWidget(ShellWidget *parent) {
+    setParent(parent);
+    hide();
```

Comment:
This can probably be done in the parent widget instead of here.

Reply:
I thought about that at first, but I wanted to put hide() in the constructor to avoid "boilerplate" in the parent widget constructor. I think it's safe to assume that all new widgets should just be hidden, and then just wait for the cmdline_show and cmdline_block_show calls

Reply:
I usually prefer this kind of things to be explicit, otherwise if someone else tries to use it, they spend some time scratching their head on why it is invisible.

Reply:
Ok, I moved the calls in `Shell` construction like for `Shell::m_pum` 
f568e67